server/service/src/invoice_line/inbound_shipment_line/generate.rs

```diff
@@ -1,6 +1,20 @@
 use repository::{InvoiceLineRow, StockLineRow};
 use util::uuid::uuid;
 
+pub fn convert_stock_line_to_single_pack(
+    stock_line: StockLineRow,
+    invoice_line: &InvoiceLineRow,
```

Comment:
Hmm any reason to pass through invoice_line, rather then just do it from stock_line ?

Reply:
Nope don't need you're correct (: